If these constants aren't defined, then the syscalls won't be built into
glibc, which will cause modutils to stop working.

--- include/asm-ia64/unistd.h.orig	2003-10-15 11:10:25.000000000 -0400
+++ include/asm-ia64/unistd.h	2003-11-04 23:48:30.000000000 -0500
@@ -121,11 +121,11 @@
 #define __NR_setdomainname		1129
 #define __NR_uname			1130
 #define __NR_adjtimex			1131
-/* 1132 was __NR_create_module */
+#define __NR_create_module		1132
 #define __NR_init_module		1133
 #define __NR_delete_module		1134
-/* 1135 was __NR_get_kernel_syms */
-/* 1136 was __NR_query_module */
+#define __NR_get_kernel_syms		1135
+#define __NR_query_module		1136
 #define __NR_quotactl			1137
 #define __NR_bdflush			1138
 #define __NR_sysfs			1139
